test/E2ETests/Common/DeploymentUtility.cs

```diff
@@ -201,7 +201,7 @@ private static Process StartMonoHost(StartParameters startParameters, ILogger lo
 
             var hostProcess = Process.Start(startInfo);
             logger.WriteInformation("Started {0}. Process Id : {1}", hostProcess.MainModule.FileName, hostProcess.Id);
-            Thread.Sleep(15 * 1000);
+            Thread.Sleep(25 * 1000);
```

Comment:
You can make requests to the server just waiting for it to response, that's a pretty common technique. This test will now *always* take 25 seconds even if the server took only 1ms to start.